v2/moviebuff.go

```diff
@@ -335,3 +336,41 @@ func (m *moviebuff) GetHolidayCalendar(ctx context.Context, countryID string) (*
 	}
 
 }
+
+func (m *moviebuff) GetLanguages(ctx context.Context) ([]Language, error) {
+	r, err := prepareRequest(ctx, m.HostURL, m.StaticToken, "/languages")
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := m.Client.Do(r)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	switch res.StatusCode {
+	case http.StatusForbidden:
+		return nil, ErrInvalidToken
+
+	case http.StatusNotFound:
+		return nil, ErrResourceDoesNotExist
+
+	case http.StatusOK:
+		content, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return nil, err
+		}
+
+		var languages []Language
+		err = json.Unmarshal(content, &languages)
+		if err != nil {
```

Comment:
```suggestion
		return languages, err
```

Reply:
Yeah if possible, for the prev function, too in L327, we could simply `return calendarInfo, nil`

Reply:
We can't do the same approach for `GetHolidayCalendar` function as it needs to return nil when there is a error. But, I'll do the change in `GetCertifications` function.

Reply:
I think we could still return with `calendarInfo`, but clients could be expecting nil when there is no data 🤔 
Anyways, that is additional things, if we are ambiguous we can ignore :)